src/nextjs-front/context/relationship/relationshipContext.ts

```diff
@@ -15,6 +23,7 @@ export type User = {
 	accountDeactivated: boolean,
```

Comment:
Above `accountDeactivated: boolean,` there is `ration` property instead of `ratio`